chromadb/server/fastapi/__init__.py

```diff
@@ -195,6 +196,134 @@ def __init__(self, settings: Settings):
 
         self.router = ChromaAPIRouter()
 
+        self.router.add_api_route("/api/v2", self.root, methods=["GET"])
+        self.router.add_api_route("/api/v2/reset", self.reset, methods=["POST"])
+        self.router.add_api_route("/api/v2/version", self.version, methods=["GET"])
+        self.router.add_api_route("/api/v2/heartbeat", self.heartbeat, methods=["GET"])
+        self.router.add_api_route(
+            "/api/v2/pre-flight-checks", self.pre_flight_checks, methods=["GET"]
+        )
+
+        self.router.add_api_route(
+            "/api/v2/auth/identity",
+            self.resolve_tenant_and_databases,
+            methods=["GET"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases",
+            self.create_database,
+            methods=["POST"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}",
+            self.get_database,
+            methods=["GET"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants",
+            self.create_tenant,
+            methods=["POST"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}",
+            self.get_tenant,
+            methods=["GET"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections",
+            self.list_collections,
+            methods=["GET"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections_count",
+            self.count_collections,
+            methods=["GET"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections",
```

Comment:
Is {database} here `database_id`? Should we name it as such if so (and other places)

Reply:
It's the database name. I can see it both ways, with a slight preference for sticking with name for user readability

Reply:
I just think we should do `database_name` or `database_id` in the code to make it clear.

User wise we should do what already works with out internal abstractions, which is `database_name`